build/templates/validate-commits.yaml

```diff
@@ -0,0 +1,10 @@
+# This template follow the Conventional Commits specification (https://www.conventionalcommits.org/en/v1.0.0/).
```

Comment:
This comment doesn't really describe the situation.

You could use this one instead.
```md
# This template is used to validate that the commit messages follow the Conventional Commits specification (https://www.conventionalcommits.org/en/v1.0.0/).
# Consider placing this at the beginning of the build pipeline to ensure that the commits are valid before proceeding with longer build steps.
```